ChemoTracker/components/SymptomTracking/Form.js

```diff
@@ -3,16 +3,21 @@ import { View, Text, StyleSheet, Dimensions } from 'react-native';
 import { StackNavigator } from 'react-navigation';
 import { LinearGradient } from 'expo';
 import Button from 'apsl-react-native-button'
+import { styles } from '../../styles/SymptomTracking/track.js';
 
 var buttonbg;
 class FormScreen extends Component {
   state = {
-    buttonbg1: 'transparent',
+    buttonbg1: '#FFF',
     buttonbg2: 'transparent',
     buttonbg3: 'transparent',
     buttonbg4: 'transparent',
+    buttontxt1: '#FB5B1B',
+    buttontxt2: '#FFF',
+    buttontxt3: '#FFF',
+    buttontxt4: '#FFF',
```

Comment:
I would suggest putting the colour names into the color file and reference {color.white}